inc/Engine/CriticalPath/Admin/Post.php

```diff
@@ -196,6 +212,10 @@ private function get_disabled_description() {
 			return '';
 		}
 
+		if ( isset( $disabled_data['not_viewable'] ) ) {
+			return __( 'This feature is not available for not public post types.', 'rocket' );
```

Comment:
Phrasing seems a bit odd here. Maybe:
```suggestion
			return __( 'This feature is not available for non-public post types.', 'rocket' );
```